masschallenge/django-accelerator
# [AC-5621] - Create Criterion object for the new Application Allocation system

To test:
check out the branch
run migrations 
run bash-shell on impact-api and examine the db tables for `accelerator_basecriterion` and `accelerator_basecriterionoptionspec`
Should see rows corresponding to the initial options and specs specified in the csv:

```
MySQL [mc_dev]> select * from accelerator_criterion;
+----+----------------------------+----------------------------+----------+----------+------------------+
| id | created_at                 | updated_at                 | type     | name     | judging_round_id |
+----+----------------------------+----------------------------+----------+----------+------------------+
|  1 | 2018-06-14 16:14:54.243759 | 2018-06-14 16:14:54.243813 | reads    | reads    |               48 |
|  2 | 2018-06-14 16:14:54.262810 | 2018-06-14 16:14:54.262965 | matching | industry |               48 |
|  3 | 2018-06-14 16:14:54.265378 | 2018-06-14 16:14:54.265414 | matching | program  |               48 |
|  4 | 2018-06-14 16:14:54.267285 | 2018-06-14 16:14:54.267319 | judge    | role     |               48 |
|  5 | 2018-06-14 16:14:54.270274 | 2018-06-14 16:14:54.270308 | judge    | gender   |               48 |
+----+----------------------------+----------------------------+----------+----------+------------------+
5 rows in set (0.01 sec)

MySQL [mc_dev]> select * from accelerator_criterionoptionspec;
+----+----------------------------+----------------------------+-----------+-------+--------+--------------+
| id | created_at                 | updated_at                 | option    | count | weight | criterion_id |
+----+----------------------------+----------------------------+-----------+-------+--------+--------------+
|  1 | 2018-06-14 16:14:54.258280 | 2018-06-14 16:14:54.258347 |           |     4 |      1 |            1 |
|  2 | 2018-06-14 16:14:54.264250 | 2018-06-14 16:14:54.264292 |           |     1 |      1 |            2 |
|  3 | 2018-06-14 16:14:54.266695 | 2018-06-14 16:14:54.266733 |           |     1 |      1 |            3 |
|  4 | 2018-06-14 16:14:54.268410 | 2018-06-14 16:14:54.268449 | Executive |     2 |      1 |            4 |
|  5 | 2018-06-14 16:14:54.269108 | 2018-06-14 16:14:54.269144 | Investor  |     1 |      1 |            4 |
|  6 | 2018-06-14 16:14:54.269705 | 2018-06-14 16:14:54.269739 | Lawyer    |     1 |      1 |            4 |
|  7 | 2018-06-14 16:14:54.271330 | 2018-06-14 16:14:54.271482 | female    |     1 |      1 |            5 |
|  8 | 2018-06-14 16:14:54.272723 | 2018-06-14 16:14:54.272764 | male      |     1 |      1 |            5 |
+----+----------------------------+----------------------------+-----------+-------+--------+--------------+
8 rows in set (0.01 sec)

MySQL [mc_dev]> select * from accelerator_basecriterionoptionspec;
ERROR 1146 (42S02): Table 'mc_dev.accelerator_basecriterionoptionspec' doesn't exist
MySQL [mc_dev]> select * from accelerator_basecriterion;
ERROR 1146 (42S02): Table 'mc_dev.accelerator_basecriterion' doesn't exist
MySQL [mc_dev]> 
```

Note that this PR does not create any of the helper classes providing functionality for the notional subtypes, that will be done in AC-5622
